src/rulesets/operations.ts

```diff
@@ -35,6 +35,7 @@ export const rules = {
       "have the correct operationId format",
       (operation, context, docs) => {
         docs.includeDocsLink(links.standards.operationIds);
+        docs.becomesEffectiveOn(new Date("2021-07-01"));
```

Comment:
Why this date instead of, say, today's date? Shouldn't anything added that requires passing this rule have been added starting from today?

I assume not actually, and that I'm missing some subtlety to how the rules around this work.

Reply:
The only resource in our API that fails this rule is our v3 Examples resource, https://apidocs.snyk.io/?version=2021-06-13%7Eexperimental#tag--Examples.

I'm being more conservative here in only exempting releases prior to 2021-07-01 from this rule. If I chose today, this might allow someone to add an operationId with a format we don't want to any active release; adding an operation is a non-breaking change.

We really should deprecate and sunset the above Examples resource from our public API.